Configurations/00-base-templates.conf

```diff
@@ -102,6 +102,8 @@ my %targets=(
                                      ? "ranlib" : "" },
         RC              => "windres",
 
+        shared_extension => ".so",
+
```

Comment:
Question: in `10-main.conf` for `darwin-common` I see:
```
shared_extension => ".\$(SHLIB_VERSION_NUMBER).dylib"
```
Which does not follow the "style" here, which looks at superficially inconsistent.  Is that a problem?

Reply:
That is perfectly correct, that's what Darwin shared library extensions looked like last I checked.  The "style" to follow is whatever is appropriate for the system the config target is for.

Reply:
s/extensions/"SONAME"/